msfrpc

```diff
@@ -83,10 +83,11 @@ rpc = Msf::RPC::Client.new(
   :ssl  => opts['SSL']
 )
 
-res = rpc.login(opts['User'], opts['Pass'])
+rpc.login(opts['User'], opts['Pass'])
 
 puts "[*] The 'rpc' object holds the RPC client interface"
-puts ""
+puts "[*] Use rpc.call('group.command') to make RPC calls"
+puts
```

Comment:
personally, I'd prefer `puts ''` to be explicit, rather than `puts` which looks like a bug.